resources/[system]/scoreboard-basic-theme/fxmanifest.lua

```diff
@@ -0,0 +1,16 @@
+-- This resource is part of the default Cfx.re asset pack (cfx-server-data)
+-- Altering or recreating for local use only is strongly discouraged.
+
+version '1.0.0'
+author 'Chip & Neco'
```

Comment:
Bundled assets should be a generic name since we're not using the fxports system at this time.